blogposts/2021/career-framework.md

```diff
@@ -0,0 +1,316 @@
+
+[Medium](https://medium.com/s/engineering-growth-framework/engineering-growth-introduction-8ba7b78c8d6c) and [Rent The Runway](https://docs.google.com/spreadsheets/d/1k4sO6pyCl_YYnf0PAXSBcX776rNcTjSOqDxZ5SDty-4/edit#gid=0) both use single-axis progression; [Buffer](https://buffer.com/resources/career-framework/) and [CircleCI](https://www.progression.fyi/f/circle-ci) both use across-the-board progression.
+
+### Designation
+
+Designation is how a company refers to different levels of ICs. The examples we looked at came in three types:
+
+- **Titles** like Senior Engineer or Staff Engineer.
+- **Numbered levels** like L2 or L3.
+- **Descriptive levels** based on scope.
+
+Spotify is an example of the last format. According to the [titles section of their career framework](https://engineering.atspotify.com/2016/02/15/spotify-technology-career-steps/#titles), an IC’s “step” (Spotify’s term for level) doesn’t map to a title. Internally, Spotify uses titles to describe an IC’s scope of impact and an IC’s given role. Externally, employees have “tremendous flexibility” for representing themselves.
+
+### Scope requirements
+
+Multiple companies we researched divided their levels by scope, meaning that as engineers advanced, the scope of their impact was expected to change.
+
+[Spotify](https://engineering.atspotify.com/2016/02/15/spotify-technology-career-steps/), for instance, matches levels directly to scope:
+
+- At the Individual Step, ICs are “focused primarily on being a useful contributor.”
+- At the Squad/Chapter Step, ICs are “a resource for your chapter or your squad.”
+- At the Tribe/Guild Step, ICs “have an impact across squads or chapters.”
+- At the Technology/Company Step, ICs focus on providing significant support for “tech-wide or company-wide initiatives.”
+
+Generally, the expectation is that as an IC advances, their impact increases in proportion with their level.
+
+## The framework roadshow: What the team had to say about work in progress
+
+The career development framework process was transparent from day one but up until the end of the project, most of the responses we got were variations of “LGTM!”
+
+It wasn't until the end, when the work was mostly done, that more substantive feedback started to come in.
+
+Even so, it wasn’t as much engagement as we would have liked. We reached out to engineering managers and asked for their feedback directly. We also paid special attention to engineers who had been at Sourcegraph a long time so as to make sure there were no concerns from the people who knew Sourcegraph best.
+
+In some cases, those feedback requests resulted in a cautious thumbs-up. Engineers liked the idea of a framework, and wanted to try what the working group had produced, but also wanted to remain open to changing the framework if it didn’t work. Luckily for them, that was the plan all along (see the **Calibration to come** section below).
```

Comment:
@rebeccadee I played around with it but couldn't figure out how to link sub-heds. Could you show me how?

Reply:
@nickmyyz There's an example in this post where it links to the video at the bottom: https://about.sourcegraph.com/blog/dev-tool-time-liz-fong-jones/

See if you can figure it out from there otherwise I'm happy to show you 😁